sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -117,7 +117,7 @@ if(SYCL_UR_USE_FETCH_CONTENT)
   endfunction()
 
   set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/UnifiedRuntimeTag.cmake)
+  set(UNIFIED_RUNTIME_TAG 698cd9974c18874bf6a7e9ec78507c30b9deeca7)
```

Comment:
I believe now you need to instead edit `UnifiedRuntimeTag.cmake` with the new commit and leave this line the same.

Reply:
ah. thank you. I missed this change.